api/tests/unit/domain/usecases/find-publishable-finalized-sessions_test.js

```diff
@@ -0,0 +1,50 @@
+const { expect, sinon, domainBuilder } = require('../../../test-helper');
+const findPublishableFinalizedSessions = require('../../../../lib/domain/usecases/find-publishable-finalized-sessions');
+
+describe('Unit | UseCase | findPublishableFinalizedSessions', () => {
+
+  let finalizedSessionRepository;
+
+  beforeEach(() => {
+    finalizedSessionRepository = {
+      findPublishableSessions: sinon.stub(),
+    };
+  });
+
+  context('when there are finalized publishable sessions', () => {
+
+    const publishableSessions = [
+      domainBuilder.buildFinalizedSession({ isPublishable: true }),
+      domainBuilder.buildFinalizedSession({ isPublishable: true }),
+      domainBuilder.buildFinalizedSession({ isPublishable: true }),
+    ];
+
+    beforeEach(() => {
+      finalizedSessionRepository.findPublishableSessions.resolves(publishableSessions);
+    });
```

Comment:
**suggestion** 
On aurait pu se passer de ces beforeEach (celui ci et le suivant) et les mettre dans les `it` pour éviter de scroller le jour où on enrichis les tests